Sources/String+Sweetness.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public extension String {
+    public var isValidEmail: Bool {
+        let emailRegEx = ".*@.*"
```

Comment:
@Elland That's another discussion, then you would have to check why we need to check valid emails at all. What I was referring to is that if we're doing it, better to do it well.